pages/osx/nvram.md

```diff
@@ -13,7 +13,7 @@
 
 - Modify the value of a firmware variable:
 
-`sudo nvram {{name}}="{{value}}"`
+`sudo nvram "{{name}}={{value}}"`
```

Comment:
are you sure?

Reply:
```bash
echo 1"2"
echo "12"
```
both output `12`. That's why I moved one quote left.

Reply:
On the other side quote after `=` looks more natural. :thinking: